### ISSUE TITLE
code for Somalia in wb destination code is missing

### ISSUE THREAD

**Issue by cjyetman** [vincentarelbundock/countrycode#203]
``` r
library(countrycode)
packageVersion("countrycode")
#> [1] '1.1.0'

countrycode("Somalia", "country.name", "wb")
#> Warning in countrycode("Somalia", "country.name", "wb"): Some values were not matched unambiguously: Somalia
#> [1] NA
```

though the [source file](http://databank.worldbank.org/data/download/site-content/CLASS.xls) used by [get_world_bank.R](https://github.com/vincentarelbundock/countrycode/blob/master/dictionary/get_world_bank.R) does contain code "SOM" for Somalia, I don't see any manual adjustments to remove it, and I don't see any previous issues here that mention Somalia.

(reported on [Stack Overflow here](https://stackoverflow.com/questions/54601932/countrycode-package-not-recognizing-som-as-iso3-character-code-for-somalia-r))

**Comment by cjyetman** [comment]
This appears to be fixed in the current dev version (1.1.3.9000). Can someone else verify?
``` r
library(countrycode)
packageVersion("countrycode")
#> [1] '1.1.3.9000'

countrycode("Somalia", "country.name", "wb")
#> [1] "SOM"
```

**Comment by vincentarelbundock** [comment]
Yes, I confirm.
